packages/minimongo/modify.js

```diff
@@ -182,6 +182,39 @@ var NO_CREATE_MODIFIERS = {
 };
 
 var MODIFIERS = {
+  $currentDate: function (target, field, arg) {
+    if (typeof arg === "object" && arg.hasOwnProperty("$type")) {
+       if (arg.$type !== "date")
+         throw MinimongoError("Minimongo does currently only support the date type in $currentDate modifiers");
+    } else if(arg !== true) {
+      throw MinimongoError("Invalid $currentDate modifier");
+    }
+    target[field] = new Date();
+  },
+  $min: function (target, field, arg) {
+    if (typeof arg !== "number")
+      throw MinimongoError("Modifier $min allowed for numbers only");
```

Comment:
Before I open an issue on this: Is there a reason why the operator isn't allowed for dates?

Reply:
Is this supported in MongoDB? Interesting.

Reply:
There's no specific reason, I wasn't aware it could be used for dates. I'll create a follow-up PR.

Reply:
Great, thanks 😊

Reply:
I looked into this a bit further and apparently $min and $max can be used to compare any types, where [BSON type order](https://docs.mongodb.com/manual/reference/bson-type-comparison-order/#faq-dev-compare-order-for-bson-types) is applied. So there's quite a bit of work involved to be fully compliant.